examples/single_tier_cache/main.cpp

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
nit. `Copyright (c) Meta Platforms, Inc. and affiliates.`

Reply:
Done